### ISSUE TITLE
doublecheck and tweak the flow of the turorial

### ISSUE THREAD

**Issue by dominikwelke** [dominikwelke/mne-python#4]
- does the general structure and logic make sense? 
- can one follow it easily?
- is there enough info in each descriptive paragraph? 
- do we need more citations? 
- does the analysis / stats example we show in the end make sense?
- ...

**Comment by kalenkovich** [comment]
Things that come to mind:

- we start slow: it takes some time to scroll through preprocessing. Suggestion: combine everything into one cell, use `verbose=False` to cut the outputs.
- epoching could be moved to preprocessing, it is not ssvep-specific.
- it is unclear why we work with SNRs instead of raw power. I think we need to add a comment on that.
- it is hard to interpret the plot with SNRs averaged over trials with one channel per line. I think the topoplot is more appropriate for that.
- I wouldn't average over all the channels - that is a good way to lose SNR and makes for noisy plots. I think we should go topoplot -> ROI (selected beforehand, note on double-dipping) -> data averaged over ROI (one line per trial, note on how we can see peaks in every trial).
- What we do in the stats section is clear, why we do what we do is less so, I think. We can figure that one out in #5 though. 

@dominikwelke, could you add what thing you think might use some updating? Then we could discuss them all together.

**Comment by dominikwelke** [comment]
- +1 for combining all preprocessing in one cell
- +1 for defining occipital roi beforehand, and then referencing this decision with the topoplot
- the full outline of the tutorial should be briefly summed up in the intro cell (so far we only have info on the data), we are missing what we want to show (corresponds to your last point)
- we could have a combined figure with the psd spectrum next to/above the snr spectrum (and with this talk a little about the differences between them, that SNR eg. adresses the 1/f thing etc). they could have the same axes and similar colorscheme
- quantification of the peaks can come after this first visualization
- it leads to the topoplot and roi subsetting cell, with discussion
- then the statistical analyses of choice can be conducted and discussed. the plot of the snr variability by trial might also fit here as an introduction?

**Comment by dominikwelke** [comment]
i'll open a branch + pr

**Comment by dominikwelke** [comment]
another option would be, to first show the average psd/snr of ALL trials, which means there would be 12hz and 15hz peaks (presumably less prominent)
not sure we should do it like this, but it might be ok if we want to walk through the data more slowly. 

possible benefits might be: 
- it could make comparing 12 and 15hz snr in the different trials easier to follow, 
- we could also compare topography of 12 and 15 hz response

**Comment by kalenkovich** [comment]
> * it could make comparing 12 and 15hz snr in the different trials easier to follow,

Yeah! We could also offer this as an alternative to a predefined roi. As long as it is not based on the comparison between the conditions, we should be good.

> * we could also compare topography of 12 and 15 hz response

Not sure I'd be able to interpret that.

**Comment by kalenkovich** [comment]
I managed to miss this PR :(

I'll go over it over the weekend, is that ok?

**Comment by kalenkovich** [comment]
> * the full outline of the tutorial should be briefly summed up in the intro cell (so far we only have info on the data), we are missing what we want to show (corresponds to your last point)

Good point.

> * we could have a combined figure with the psd spectrum next to/above the snr spectrum (and with this talk a little about the differences between them, that SNR eg. adresses the 1/f thing etc). they could have the same axes and similar colorscheme

Yeah, makes sense as the justification for preferring snrs.

> * quantification of the peaks can come after this first visualization

Not sure what you mean by "quantification of the peaks".

> * it leads to the topoplot and roi subsetting cell, with discussion

> * then the statistical analyses of choice can be conducted and discussed. the plot of the snr variability by trial might also fit here as an introduction?

Yeah, we can plot the data itself in addition to doing a test.

**Comment by kalenkovich** [comment]
I suggest we link the PR to this issue (adding "closes #4" in the first comment of the PR) and add a checkbox list in the first cell of this issue. It will help us to keep track of what we decide to do. Would that be alright?

**Comment by kalenkovich** [comment]
Yeah, I got confused with github's cross-referencing of issues/prs. Of course, the checklist should go into PR. Let's add/remove points there as we discuss this. I've already added the points that I think we agree on.
